openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1449,6 +1451,16 @@ Access Tokens represent End-User authorization and consent to issue certain Cred
 
 To sender-constrain Access Tokens, see the recommendations in (#securitybcp). If Bearer Access Tokens are stored by the Wallet, they MUST be stored in a secure manner, for example, encrypted using a key stored in a protected key store.
 
+## Application-Layer Encryption {#encryption-security-considersations}
+
+Depending on the architecture of the Wallet and the Issuer, additional encryption of requests and responses may provide additional confidentiality for data beyond TLS, at the cost of increased complexity. An example is when a Wallet is composed of different components with differing levels of trust (such as a Wallet server backend and client application). The same may apply to complex Issuer systems.
+
+It is important that the component performing the encryption on the Wallet is able to establish trust in the Issuer key material being used to perform the encryption, to prevent being man-in-the-middled. The simplest way to do this is to retrieve it directly from the Issuer-hosted Issuer Metadata. Another is to verify the signature in the case of signed Issuer Metadata.
```

Comment:
```suggestion
It is important that the Wallet component responsible for encryption can establish trust in the Issuer's key material to prevent man-in-the-middle attacks. The simplest way to achieve this is by retrieving the keys directly from the Issuer’s hosted Issuer Metadata. Alternatively, in the case of signed Issuer Metadata, the signature can be verified to ensure authenticity.
```